src/resources/css/rsvp.css

```diff
@@ -187,4 +187,18 @@
 .event-tickets-meta-label {
 	font-weight:700;
 	margin: 0 1em 0 0;
-}
\ No newline at end of file
+}
+
+/* TwentySixteen fix for box shadow as bottom border on <a> link */
+
+.entry-content a,
+.entry-summary a,
+.taxonomy-description a,
+.logged-in-as a,
+.comment-content a,
+.pingback .comment-body > a,
+.textwidget a,
+.entry-footer a:hover,
+.site-info a:hover {
+	box-shadow: none;
+}
```

Comment:
Hmm, guess it's a bit of a judgement call. Re copycats - though it wouldn't cover _all_ scenarios - we could provide a little more coverage by also using `.tribe-theme-parent-{theme}` (then child themes of Twenty Sixteen would also benefit).